compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/type/builders/BallerinaTableTypeBuilder.java

```diff
@@ -156,9 +158,13 @@ private BType getKeyConstraintBType(List<TypeSymbol> keyTypes, TypeSymbol rowTyp
             return checkKeyConstraintBType(keyTypes.get(0), rowType);
         }
 
-        List<BType> tupleMemberTypes = new ArrayList<>();
+        List<BTupleMember> tupleMemberTypes = new ArrayList<>();
```

Comment:
Isn't this 
```suggestion
        List<BTupleMember> tupleMembers = new ArrayList<>();
```

As suggested previously, please check and fix wherever applicable.